docs/settings.rst

```diff
@@ -36,8 +36,8 @@ The following are known to be effective:
 - ``SCHEMA_COERCE_PATH_PK``
 
 
-Example: API Key securitySchemes & security setting
----------------------------------------------------------------------
+Example: API Key securitySchemes & security setting & SwaggerUI Config
```

Comment:
please open up a new section for documenting those settings. mixing swagger with security settings may be confusing. just take your doc and do a new `Example: SwaggerUI Config`